### PR TITLE
Fix #20096: Server list window scroll is glitched when server list is empty

### LINKED ISSUE #20096: Server list window scroll is glitched when server list is empty
### Operating System

Windows 10

### OpenRCT2 build

OpenRCT2 fe219281ec6b66e3a47c67125c939bc1f01d1274

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

When the server list is first opened, the scroll is glitched. This only works in OpenGL mode. My graphic card is a Nvdia GTX 1060.

image

### Steps to reproduce

1. Start OpenRCT2 in OpenGL mode
3. Click on Play with other players

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by beam41**
Scroll widget have incorrect size set, plus cast to uint16_t makes it underflow.

~~Actually there are some more scrollbar with incorrect size. like one in select scenario which update when you reopen.~~
I'm also changed every scroll that have incorrect size.
image
image

Fix #20096

### CODE REVIEW

File: src/openrct2/interface/Window.cpp [hunk 1]
```diff
@@ -2246,10 +2246,10 @@ void WidgetScrollUpdateThumbs(WindowBase& w, WidgetIndex widget_index)
 
         if (scroll.v_thumb_bottom - scroll.v_thumb_top < 20)
         {
-            double barPosition = (scroll.v_thumb_bottom * 1.0) / view_size;
+            double barPosition = ((scroll.v_thumb_bottom * 1.0) / view_size);
```
Comment (ZehMatt) [inline src/openrct2/interface/Window.cpp:2249]: Unrelated change, please revert this.

**Review by ZehMatt — CHANGES_REQUESTED** [review]
I've tested it and it all seems fine, left a comment otherwise all good.

File: src/openrct2/interface/Window.cpp (hunk 1 above)
Comment (frutiemax) [reply src/openrct2/interface/Window.cpp:2249]: Why use double over float?

**Review by ZehMatt — APPROVED** [review]
Thanks, all good. Could you add a changelog entry as this bug was introduced in 0.4.4

**Comment by beam41** [comment]
> Thanks, all good. Could you add a changelog entry as this bug was introduced in 0.4.4

Done

**Comment by ZehMatt** [comment]
Sorry but could you rebase the PR, we just merged a few things that changed changelog.txt

**Pull request merged**